kuma/wiki/views/list.py

```diff
@@ -123,6 +130,7 @@ def with_errors(request):
     return render(request, 'wiki/list/documents.html', context)
 
 
+@block_user_agents
```

Comment:
Should we add https://developer.mozilla.org/en-US/docs/without-parent to robots.txt?